### DISCUSSION TITLE
How to improve entity availability? 24h off

### DISCUSSION THREAD

**Discussion by tom-winkler** [Thomas55555/husqvarna_automower#181] [Q&A]
![Screenshot_20220316-205040_Home Assistant](https://user-images.githubusercontent.com/9554467/158678297-c7d760c1-31e3-4b9b-91ca-d17d43d516cf.jpg)
Any clue how to get a recent state update even if the device (415x in my case) is not mowing?

**Comment by simontegelid** [comment]
The websocket API should update the mower even when it's not mowing, but it's rarely needed though since its not mowing. We could add sporadic polls as well, apart from websocket updates. I will shift my focus back to this integration as soon as my lawn is not covered in snow any longer :)

**Reply by tom-winkler** [reply]
No worries can do this either....maybe also play around in ha to configure timeout.

**Reply by Thomas55555** [reply]
I don't understand the problem exactly. If the mower is not available there shouldn't be an update in the state machine, because it stays unavailable. But I've included polls, if the websocket is not updating anymore. You can try version 2022.4.1b1, if this helps for your problem. Please give me a feedback, if it works.

**Comment by Thomas55555** [comment]
Btw. Have you ever tested, if changing the cutting height is working? I just coded it, but I don't have a possibility to test.

**Reply by astrandb** [reply]
> Btw. Have you ever tested, if changing the cutting height is working? I just coded it, but I don't have a possibility to test.

Yes, it works fine on my 430X
